app/views/shared/_terms.html.erb

```diff
@@ -520,73 +520,123 @@ http://www.michigan.gov/disabilityresources/
-access these services.</p>
+275-6424</p>
+<p>
+  Citizens and Non-Citizens /Social Security Numbers
+</p>
+<p>
+  Social Security numbers and immigration papers are NOT required for a person who
+  is:
+</p>
+
+<ul>
+  <li>
+    Not applying for help.
+  </li>
+  <li>
+    An undocumented non-citizen only applying for medical assistance for emergency
+    services, pregnancy or child birth.
+  </li>
+  <li>
+    Only applying for child care. (You must give a Social Security number for the
+    child and the child must be a U.S. citizen or show immigration papers.)
+  </li>
+</ul>
+<p>
+Other eligible members of your household will still be able to receive help.
+</p>
+<p>
+  You may have to provide information about income and assets of all persons in
+  your household, even if they are not applying.
+</p>
+<p>
+  Receiving food, medical, or emergency assistance will not affect your
+  immigration status. If you are here illegally, it may affect your ability to
+  stay in the U.S.
+</p>
+<p>
+  For some programs, persons claiming U.S. citizenship must provide proof of
+  citizenship and identity. Acceptable proof of citizenship includes, but is not
+  limited to, a U.S. passport, a certificate of naturalization, a U.S. public
+  birth record showing birth in the U.S. or U.S. territories.
+</p>
+<p>
+  Persons receiving SSI, Social Security, Medicare, or adoption assistance; foster
+  children, and newborn "safe delivery" babies are not required to provide proof
+  of U.S. citizenship for MDHHS programs.
+</p>
+<p>
+  Race and Ethnicity
+</p>
+<p>
+  Answering questions about race and ethnicity is voluntary. If you do not answer
+  these questions, your eligibility or benefit levels will not be affected.* The
+  information is collected to ensure that program benefits are distributed without
+  regard to race, color or national origin.
+</p>
+<p>
+  *If you choose not to answer these questions, your MDHHS specialist may choose
+  an answer for you.
+</p>
+
+<p>
+  Domestic Violence
+</p>
+
+<p>
+  We may be able to waive some program requirements (such as working, looking for
+  a job, pursuing child support or going to school) if participating would:
+</p>
+
+<ul>
+  <li>Put you or a family member in danger of physical or emotional harm.</li>
+  <li>Subject you to sexual abuse.</li>
+  <li>Otherwise be unfair to you.</li>
+</ul>
+
+<p>
+  You are authorized to receive domestic violence comprehensive services. Contact
+  the MDHHS office in your area or your MDHHS specialist for more information or
+  to access these services.
+</p>
 
 <p>Resources:</p>
 
 <ul>
-  <li >Online at: <a href="http://www.michigan.gov/domesticviolence>www.michigan.gov/domesticviolence"</a></li>
-  <li >DHS Publication 859, Is Someone Hurting You or Your Children? (also available in Spanish) -</li>
+  <li>
+    Online at: <a
+      href="http://www.michigan.gov/domesticviolence">www.michigan.gov/domesticviolence"</a>
```

Comment:
Dangling `"` before `</a>`.

Reply:
oh yeah that's in an attempt to make the lines shorter :)

Reply:
woops, didnt read your comment correctly

fixed at 19db043f